source/style/_theme/sidebar.less

```diff
@@ -142,11 +166,6 @@
 				max-height: 0;
 				overflow: hidden;
 			}
-
-			li {
-				@indent: 15px;
-				//padding-left: @indent;
```

Comment:
This previous `padding-left` was pretty important, in that it maintained the nested structure which was implied by the "unordered lists" (`<ul>`s).  This has been re-created by use of the nested `li` classing shown above and to the right.

Reply:
Wow, thank goodness, I didn't remove this commented section just yet. Great job @peggy @abernix